iam/api-client/src/main/java/com/google/iam/snippets/ServiceAccountKeyCreate.java

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.iam.snippets;
+
+// [START iam_create_key]
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.iam.v1.Iam;
+import com.google.api.services.iam.v1.IamScopes;
+import com.google.api.services.iam.v1.model.CreateServiceAccountKeyRequest;
+import com.google.api.services.iam.v1.model.ServiceAccountKey;
+import java.util.Collections;
+
+
+public class ServiceAccountKeyCreate {
+
+    public static ServiceAccountKey createKey(String serviceAccountEmail) throws Exception {
+
+        GoogleCredential credential = GoogleCredential.getApplicationDefault()
+        .createScoped(Collections.singleton(IamScopes.CLOUD_PLATFORM));
+
+        Iam service = new Iam.Builder(GoogleNetHttpTransport.newTrustedTransport(), JacksonFactory.getDefaultInstance(),
+            credential).setApplicationName("service-account-keys").build();
+        
+        ServiceAccountKey key = service.projects().serviceAccounts().keys()
+            .create("projects/-/serviceAccounts/" + serviceAccountEmail, new CreateServiceAccountKeyRequest()).execute();
+    
+        System.out.println("Created key: " + key.getName());
+        return key;
```

Comment:
We want to avoid returning anything, and instead just output content to the console.

Reply:
I'm struggling with how to do this, since I need to use the created objects (service account/service account key) to test the other methods. Suggestions?

Reply:
What you want to do it create a test fixture - you can use the `@Before` (and `@After`)methods to create (and then delete) a resource for each test. 

Very rarely, it's too expensive to create a resource for each test. In this case you would want to use `@BeforeClass` and `@AfterClass` to create/delete a shared resource for the entire class.

Reply:
Done